From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rene Hickersberger <r@renehsz.com>
Date: Sat, 24 Jun 2023 01:15:26 +0200
Subject: [PATCH] Remove needless include of ftw.h

This header does not exist on Serenity, but it seems to be unused.
---
 src/nnn.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/nnn.c b/src/nnn.c
index b3c0f986ff843b80b0e3c349ac70e151cc9de844..76833f3e08d0a0abfa22ae65cb381855e85fd048 100644
--- a/src/nnn.c
+++ b/src/nnn.c
@@ -105,7 +105,6 @@
 #ifndef __USE_XOPEN_EXTENDED
 #define __USE_XOPEN_EXTENDED 1
 #endif
-#include <ftw.h>
 #include <pwd.h>
 #include <grp.h>
 
